http_client.rst

```diff
@@ -728,6 +728,39 @@ When using this component in a full-stack Symfony application, this behavior is
 not configurable and cURL will be used automatically if the cURL PHP extension
 is installed and enabled. Otherwise, the native PHP streams will be used.
 
+Providing additional options to CurlHttpClient
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to provide additional cURL options to CurlHttpClient. PHP exposes
+a lot of `cURL options`_ that can be passed to ``curl_setopt`` function, but only some
+of them are used in CurlHttpClient in favor of bigger component portability.
+
+To provide cURL-related parameters to request, add an ``extra.curl`` option in your
+configuration::
+
+    use Symfony\Component\HttpClient\CurlHttpClient;
+
+    $client = new CurlHttpClient();
+
+    $client->request('POST', 'https://...', [
+        // ...
+        'extra' => [
+            'curl' => [
+                \CURLOPT_IPRESOLVE => \CURL_IPRESOLVE_V6
```

Comment:
IMO we can remove the leading slashes to keep the example easier to read

Reply:
fixed.

Reply:
fixed.